### PR TITLE
Add export/import of the whole project

### PR DESCRIPTION
**Pull request by liry**
Implements #965

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-java-model/src/main/java/com/gooddata/sdk/model/md/maintenance/ExportProjectArtifact.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.gooddata.sdk.model.gdc.UriResponse;
+
+/**
+ * Complete project export result structure.
+ * For internal use only.
+ * Deserialization only.
+ */
+@JsonTypeName("exportArtifact")
+public class ExportProjectArtifact extends PartialMdArtifact {
```
Comment (peter-plochan) [inline gooddata-java-model/src/main/java/com/gooddata/sdk/model/md/maintenance/ExportProjectArtifact.java:19]: Hmm, we should reconsider if these internal classes are part of the model or not.

**Review by peter-plochan — CHANGES_REQUESTED** [review]

File: gooddata-java-model/src/main/java/com/gooddata/sdk/model/md/maintenance/ExportProjectArtifact.java (hunk 1 above)
Comment (liry) [reply gooddata-java-model/src/main/java/com/gooddata/sdk/model/md/maintenance/ExportProjectArtifact.java:19]: Well, this one is not really internal, it's part of the API, so I would say that it should be in the model (one may want to reuse it for API implementation).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-java-model/src/main/java/com/gooddata/sdk/model/md/maintenance/ExportProject.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2004-2019, GoodData(R) Corporation. All rights reserved.
```
[the remark this answers is not in the document]
Comment (liry) [reply gooddata-java-model/src/main/java/com/gooddata/sdk/model/md/maintenance/ExportProject.java]: copyright updated in all touched files

File: gooddata-java-model/src/main/java/com/gooddata/sdk/model/md/maintenance/ExportProjectToken.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2004-2019, GoodData(R) Corporation. All rights reserved.
```
[the remark this answers is not in the document]
Comment (liry) [reply gooddata-java-model/src/main/java/com/gooddata/sdk/model/md/maintenance/ExportProjectToken.java]: copyright updated in all touched files

**Review by peter-plochan — APPROVED** [review]

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-java-merge-compliance-check/183/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-java-fossa-scanning-zuul-el7/23/ : SUCCESS in 13s (non-voting)

**Pull request merged**